src/reviews/README.md

```diff
@@ -17,46 +17,56 @@ There are several time periods during the whole Call for Proposals and Proposal
 6. Internal Decision
 7. Announcement of Acceptance
 
-Each time period has a set of settings that need to be set in src/pycontw2016/settings, or
+Each time period has a set of settings that need to be set in the Django administration dashboard, or
 some steps you might need to go through, which will be documented here.
 
 ### 1. Call for Proposals
 
-`settings.py`:
+In the administration dashboard, go to `Home > Django Registry > Entries` and you should have the following variables
```

Comment:
How about also providing URL like `https://localhost:8000/2020/admin/registry/entry/`

Reply:
Good advice, but I think in the development environment, the year is not necessary? maybe something like `https://localhost:8000/admin/registry/entry/`? need to test it out.

Reply:
yep, confirmed, in local development environment, the `2020` is not required. But I'm curious does it vary in different environment? Can you check for us @Lee-W :D

Reply:
I incline to keep the description yearly-neutral. Your suggestion could be an example. What do you think?

Reply:
Fixed up according to your suggestion. You may want to review it again.